corona-warn-app/cwa-app-android
# Final deletion date for Recycler Bin (EXPOSUREAPP-10766)

This PR adds the final deletion date to the recycler bin items.

How to test:
- delete some certificates
- go to the recycler bin inside the menu
- check if the deletion date is shown and correct

Screenshot:
![image](https://user-images.githubusercontent.com/46747780/148575165-a450f81e-987f-497a-890d-47bb6f64b740.png)
